src/neptune/new/__init__.py

```diff
@@ -36,6 +36,7 @@
     "init_project",
     "init_run",
     "Run",
+    "RunType",
```

Comment:
Let's move it to `new/integrations/utils.py` instead.

Reply:
But it isn't strictly related to integrations. 

Moreover, my thinking was to put it close to Run so in case it is refactored, it is harder to miss the type signature.

Reply:
> Since this is for internal use, I assumed that there is no need to update CHANGELOG for that.

It isn't internal as it was added to the most exposed module :P. 

> But it isn't strictly related to integrations.

What is an example of something that is not the integration but is still internal?

Reply:
> Moreover, my thinking was to put it close to Run so in case it is refactored, it is harder to miss the type signature.

Even it's close it doesn't mean that will be updated. I think IDE will handle that and no reason to concern

Reply:
Done